crates/matrix-sdk/src/room/mod.rs

```diff
@@ -3021,6 +3021,26 @@ impl Room {
         Ok(self.client.send(request).await?)
     }
 
+    /// Reports a room as inappropriate to the server.
+    /// The caller is not required to be joined to the room to report it.
+    ///
+    /// # Arguments
+    ///
+    /// * `reason` - The reason the room is being reported.
+    ///
+    /// # Errors
+    ///
+    /// Returns an error if the room is not found or on rate limit
+    pub async fn report_room(
```

Comment:
I think the method name should be `report`, thoughts?

Reply:
Could be ambiguous with respect to reporting a user?

Reply:
Hmm indeed. Let's keep `report_room` then.